### PR TITLE
Fixed rubocop and foodcritic for yum-osuosl

### PR DESCRIPTION
**Pull request by cthalmann**
Chefspec tests need to be added to yum-osuosl. In order to pass rake, this cookbook has to pass rubocop and foodcritic before even getting to chefspec. This PR fixes rubocop and foodcritic errors. I also used chef generate in order to add necessary files.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+DEBIAN_8 = {
+  platform: 'debian',
+  version: '8.4'
+}.freeze
+
+ALL_PLATFORMS = [
+  CENTOS_6,
+  CENTOS_7,
+  DEBIAN_8
```
Comment (ramereth) [inline spec/spec_helper.rb]: Remove Debian references. This is only for CentOS.

**Comment by ramereth** [comment]
@cthalmann can you please update ``.kitchen.yml`` to our standards and remove ``.kitchen.cloud.yml``?

**Comment by ramereth** [comment]
@cthalmann can you please add CentOS 7 to supports in metadata and also remove the double quotes?

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Metrics/LineLength:
+  Max: 120
+
+Style/IfUnlessModifier:
+  MaxLineLength: 120
+
+Style/WhileUntilModifier:
+  MaxLineLength: 120
+
+Metrics/BlockLength:
```
Comment (fahlmant) [inline .rubocop.yml]: Not needed.

File: metadata.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@
 long_description 'Installs/Configures yum-osuosl'
 version          '0.1.1'
 
-supports "centos", "~> 6.0"
+supports 'centos', '~> 6.0'
+supports 'centos', '~> 7'
```
Comment (ramereth) [inline metadata.rb]: This needs to be ``7.0`` not ``7``.

**Comment by cthalmann** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.1.2.  Have a nice day!

**Pull request merged**